### ISSUE TITLE
Keyboard navigation doesn’t work as expected when using RTL

### ISSUE THREAD

**Issue by jouni** [vaadin/vaadin-tabs#105]
This applies to horizontal tabs only. The left and right arrow keys should move the focus in the opposite direction when using RTL.

**Comment by web-padawan** [comment]
Will be ixed by vaadin/vaadin-list-mixin#66. Already fixed for the `next` branch.
